src/core/include/openvino/op/squeeze.hpp

```diff
@@ -11,11 +11,43 @@
 
 namespace ov {
 namespace op {
+namespace util {
+/// \brief Squeeze operation.
+///
+/// \ingroup ov_ops_cpp_api
+class OPENVINO_API SqueezeBase : public Op {
```

Comment:
Move this class to:
`src/core/include/openvino/op/util/squeeze_base.hpp`

Reply:
Done